htp/htp_connection_parser.c

```diff
@@ -194,8 +194,6 @@ htp_tx_t *htp_connp_tx_create(htp_connp_t *connp) {
     htp_tx_t *tx = htp_tx_create(connp);
     if (tx == NULL) return NULL;
 
-    connp->in_tx = tx;   
-
     htp_connp_in_reset(connp);
```

Comment:
Do we still need this ?

Reply:
Just so I understand correctly, are you talking about the call to htp_connp_in_reset()?

Reply:
Indeed